src/typings/questionnaire.tsx

```diff
@@ -89,6 +87,10 @@ export interface BaseQuestionElement {
     isRequired?: boolean;
 }
 
+export interface BaseQuestionElement extends BaseQuestionElementWithoutId {
```

Comment:
We do not need to create BaseQuestionElementWithoutId. We can use Omit utility type https://www.typescriptlang.org/docs/handbook/utility-types.html#omittk